modules/api/docs/README.md

```diff
@@ -0,0 +1,21 @@
+# @luma.gl/api
+
+luma.gl provides an abstract API for writing code that works with both WebGPU and WebGL.
+
+The API is modeled after the WebGPU API but is stream-lined to be less cumbersome to use.
+
+## Installing adapters
+
+The `@luma.gl/api` module is not usable on its own. A device adapter must be registered.
+
+```typescript
+import {luma} from '@luma.gl/api';
```

Comment:
This will be confusing down the road since we also have a global variable `luma` for log level etc.

Reply:
Fair point. We have some time to decide what to call this global variable.